src/commands/core/follow.ts

```diff
@@ -14,6 +14,9 @@ export default class extends Command {
                 permission: 'Manage Webhooks'
             }));
 
+        const webhooks = await message.channel.fetchWebhooks();
+        if (webhooks.size === 10) return message.error(message.translate('core/follow:MAX_WEBHOOK_LIMIT'));
```

Comment:
```suggestion
        if (webhooks.size >= 10) return message.error(message.translate('core/follow:MAX_WEBHOOK_LIMIT'));
```

Reply:
Makes this a bit more flexible and less prone to errors if someone somehow has more than 10 webhooks.